packages/standard-components/src/forms/InnerForm.svelte

```diff
@@ -108,8 +116,28 @@
 
       return !newError
     }
+
+    const clearValue = () => {
+      const { fieldState } = fieldMap[field]
+      fieldState.update(state => {
+        state.value = defaultValue
+        state.error = null
+        return state
+      })
+
+      formState.update(state => {
+        state.values = { ...state.values, [field]: defaultValue }
+        delete state.errors[field]
+        state.valid = Object.keys(state.errors).length === 0
+        return state
+      })
+
+      return true
```

Comment:
We don't need to return anything here since we never need the result of this function, we could just remove this line.